frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/FieldListDrawer/FieldListOption.tsx

```diff
@@ -69,38 +71,42 @@ export const DraggableBasicFieldListOption = ({
   children,
   isDisabled,
   ...props
-}: DraggableBasicFieldOptionProps): JSX.Element => (
-  <Draggable
-    index={index}
-    isDragDisabled={isDisabled}
-    disableInteractiveElementBlocking
-    draggableId={fieldType}
-  >
-    {(provided, snapshot) => {
-      return (
-        <>
-          <BasicFieldOption
-            fieldType={fieldType}
-            isDisabled={isDisabled}
-            {...props}
-            {...provided.draggableProps}
-            {...provided.dragHandleProps}
-            style={getDraggableAnimationProps(provided, snapshot)}
-            ref={provided.innerRef}
-          />
-          {snapshot.isDragging && (
+}: DraggableBasicFieldOptionProps): JSX.Element => {
+  const isMobile = useIsMobile()
+
+  return (
+    <Draggable
+      index={index}
+      isDragDisabled={isDisabled || isMobile}
```

Comment:
we could potentially also implement on drag = close builder drawer for mobile to enable this feature but maybe after react :P